interactive-clap-derive/src/debug.rs

```diff
@@ -0,0 +1,22 @@
+#[cfg(feature = "introspect")]
+macro_rules! dbg_cond {
+    ($($val:expr),* ) => {
+        dbg!($($val),*)
+    };
+}
+
+/// this macro under `introspect` feature can be used to debug how derive proc macros
+/// ([`crate::InteractiveClap`], [`crate::ToCliArgs`]) work
+///
+/// ```bash
+/// # interactive-clap-derive folder
+/// cargo test test_doc_comments_propagate --features introspect -- --nocapture
+/// # from repo root
+/// cargo run --example struct_with_subargs --features interactive-clap-derive/introspect
+/// ```
+#[cfg(not(feature = "introspect"))]
+macro_rules! dbg_cond {
+    ($($val:expr),*) => {
+        // no-op
```

Comment:
I think you can use allow unused here.

and then you won't need to #[allow(clippy::unused_enumerate_index)] in other files

Reply:
and then this popped up too https://github.com/near-cli-rs/interactive-clap/pull/26/commits/bc0a3639499dc353cfa6c6a9416a628b8c890a74

Reply:
eventually getting rid of `#[allow(clippy::unused_enumerate_index)]` could be done, but it required
returning an unused expression. Probably compiler will optimize it out anyway, so it's not much of a big deal